infra/gcb/build_project.py

```diff
@@ -324,6 +333,9 @@ def get_build_steps(project_dir):
                     targets_list_url,
                 ],
             },
+            # upload the latest.version file
```

Comment:
nit: Capitalize comment and end with period.

Reply:
consistency with the other comments!!! (sorry I've clicked merged literally the same second your comments popped up)